README.md

```diff
@@ -59,54 +59,56 @@ npm ci
 make serve
 ```
 
-This will start the local Hugo server on port 1313. Open up your browser to http://localhost:1313 to view the website. As you make changes to the source files, Hugo updates the website and forces a browser refresh.
+This will start the local Hugo server on port 1313. Open up your browser to <http://localhost:1313> to view the website. As you make changes to the source files, Hugo updates the website and forces a browser refresh.
 
 ## Building the API reference pages
 
-The API reference pages located in `content/en/docs/reference/kubernetes-api` are built from the Swagger specification, using https://github.com/kubernetes-sigs/reference-docs/tree/master/gen-resourcesdocs.
+
+The API reference pages located in `content/en/docs/reference/kubernetes-api` are built from the Swagger specification, using <https://github.com/kubernetes-sigs/reference-docs/tree/master/gen-resourcesdocs>.
 
 To update the reference pages for a new Kubernetes release (replace v1.20 in the following examples with the release to update to):
 
 1. Pull the `kubernetes-resources-reference` submodule:
 
-```
-git submodule update --init --recursive --depth 1
-```
+    ```bash
+    git submodule update --init --recursive --depth 1
+    ```
```

Comment:
1. If you want to change the English version, put the change in a separate PR;
1. Code snippets like this should be indented by three spaces, i.e. left aligned to the first 'P' character on line 71. Please avoid indent by 4 spaces which has a special meaning in markdown.

Reply:
> 1. If you want to change the English version, put the change in a separate PR;
> 2. Code snippets like this should be indented by three spaces, i.e. left aligned to the first 'P' character on line 71. Please avoid indent by 4 spaces which has a special meaning in markdown.

ok